server/utils/nunjucksSetup.ts

```diff
@@ -52,5 +52,33 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
     return dayjs(date).format('D MMM YYYY')
   })
 
+  njkEnv.addFilter('calculateDays', (date: string) => {
+    const appt = dayjs(date).format('D MMM YYYY')
+    const today = dayjs().format('D MMM YYYY')
+
+    const diffInDays = dayjs(appt).diff(today, 'day')
```

Comment:
We also need to consider business days, I think weekends and holidays aren't considered in the diff (unless this requirement is a different ticket, I see you have stuck to the table in the ticket here)

Reply:
not relevant to this ticket I don't think: https://dsdmoj.atlassian.net/browse/WFP-669
Maybe we should do PR reviews over slack?

Reply:
Yeah, I was just looking at the requirements for this ticket, business days  are mentioned in 756